.gitignore

```diff
@@ -47,3 +47,4 @@ source-css
 
 
 dist
+.vscode/settings.json
```

Comment:
I don't think we want to ignore settings.json do we as this will have linting rules/config etc?